### ISSUE TITLE
Quick-open script Ctrl+Alt+O no longer works in the script editor

### ISSUE THREAD

**Issue by Zylann** [godotengine/godot#43117]
Godot 3.2.4 beta1
Windows 10 64 bits
Using qwerty keyboard

In Godot 3.2.3 and before there is a shortcut to open any script with fuzzy search by using Ctrl+Alt+O:
![image](https://user-images.githubusercontent.com/1311555/97249443-bcff2500-17fb-11eb-8ff0-ea6679def755.png)
This shortcut worked from anywhere: 2D editor, 3D editor, **script editor** and even the shader editor.

Now it no longer works in the script editor and the shader editor, and instead prints a `ó`.

**Comment by volzhs** [comment]
having same issue on Kubuntu 20.04

**Comment by EricEzaM** [comment]
Unintended consequence of #37769 I think.

The code in that PR is called on _gui_input(), and the event is then accepted so it never makes it to the shortcut handler.

**Comment by vnen** [comment]
This is also a problem in master (at 613b76cfd5b57d6fb13f856ad149f2b47ca5dcb3).

**Comment by akien-mga** [comment]
I can confirm it on `master` and `3.2` too. When you're used to it it's quite a workflow disruption (it still works if you click outside the script editor to force the focus away), would be nice to figure out a fix.

**Comment by EricEzaM** [comment]
90% sure #37769 is what causes this. In the linked issue it was said that issue did not occur if Alt Gr was used... is that enough, or is this fix still needed?

Not sure how to fix... TextEdit/LineEdit will 100% always try to handle the input first, and if it is handled then shortcuts will never try to be triggered. So a) remove the fix from that PR, or b) can't use control+alt shortcuts?

**Comment by Cykyrios** [comment]
I've noticed this issue as well (I'm using Linux Mint 20), actually my issue is mainly with the Ctrl+Alt+S shortcut to save a script, which is *really* disrupting as I have to instead right click in the script list area to save. I can confirm #37769 is the source of this issue as I have bisected it to commit f412237.
It seems this was originally a Windows-related issue, should a fix be made to differentiate between Windows and Linux? I see there are some `#ifdef APPLE_STYLE_KEY` in the code close to the affected lines, so more could be added. If the fix appears to be easy enough, I will make a pull request later today (I should be able to test the changes on both Linux and Windows).
